.rubocop.yml

```diff
@@ -113,6 +113,12 @@ RSpec/NotToNot:
 RSpec/Rails/HttpStatus:
   EnforcedStyle: numeric
 
+Style/ClassAndModuleChildren: # Do what makes sense in context
```

Comment:
Looking at the rule doc, I wonder if there it is actually mostly `EnforcedStyle: compact`. I'll take a look later
```suggestion
# Reason:
# https://docs.rubocop.org/rubocop/cops_style.html#styleclassandmodulechildren
Style/ClassAndModuleChildren:
```